ui/src/reusable_ui/components/card_select/CardSelectCard.tsx

```diff
@@ -1,86 +1,59 @@
 // Libraries
 import React, {PureComponent} from 'react'
 import classnames from 'classnames'
-
-// Types
-import {CardSelectCardProps} from 'src/types/cardSelect'
-
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
-interface State {
-  checked: boolean
+interface Props {
+  id: string
+  name?: string
+  label: string
+  image?: string
+  checked?: boolean
+  disabled?: boolean
+  onClick: () => void
 }
 
 @ErrorHandling
-class CardSelectCard extends PureComponent<CardSelectCardProps, State> {
-  public static defaultProps: Partial<CardSelectCardProps> = {
-    checked: false,
-    disabled: false,
```

Comment:
I actually really like it when if a prop is not required, there is a default explicitly defined for it. I think it makes it much easier for someone to reason about or change default behavior- and if not the methods end up being responsible for default behavior- and that's not their responsibility. What do you think?